odd-platform-api/src/test/java/org/opendatadiscovery/oddplatform/repository/NamespaceRepositoryImplTest.java

```diff
@@ -0,0 +1,68 @@
+package org.opendatadiscovery.oddplatform.repository;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opendatadiscovery.oddplatform.BaseRepositoryTest;
+import org.opendatadiscovery.oddplatform.misc.TestDataUtils;
+import org.opendatadiscovery.oddplatform.model.tables.pojos.NamespacePojo;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Integration test for {@link NamespaceRepository}
+ *
+ * @author matmalik on 10.12.2021
+ */
+class NamespaceRepositoryImplTest extends BaseRepositoryTest {
+
+    @Autowired
+    private NamespaceRepository namespaceRepository;
+
+    TestDataUtils testDataUtils = new TestDataUtils();
+
+    /**
+     * Tear down database entries
+     */
+    @BeforeEach
+    void tearDown() {
+        namespaceRepository.deleteAll();
```

Comment:
It's just a suggestion, but you can use @Sql annotation and define script which will clear database before each test. Thus it won't be necessary to add deleteAll() method to repository, which is not used in our business code (and which can be dangerous)